update install with proguard conf and transitive dependency (#50)
miamtech/mealz-documentation: docs/android/overview/installation.md

}
```

then import in your module build.gradle 

``` gradle
// Your module gradle file
implementation("ai.mealz.android:mealz-android:version")
```

## ProGuard config

> If you have enable minification, you'll have to add those ligne in your ProGuard config file 

Reviewer: ```suggestion
If you have enabled minification, you must add these lines in your ProGuard config file 
```
